Dockerfile

```diff
@@ -3,6 +3,9 @@ FROM ubuntu:eoan
 # install legacy build environment for docs
 RUN apt-get -o Acquire::Check-Valid-Until=false update
 RUN apt-get -y install libpython2.7-dev python2.7 git python-pip rsync
+RUN apt-get -y install curl
+RUN curl https://bootstrap.pypa.io/get-pip.py --output get-pip.py
+RUN python2 get-pip.py
 RUN pip install requests virtualenv virtualenvwrapper py-dateutil
```

Comment:
It's always better to specify the python interpreter when calling pip, e.g. `python2 -m pip install ...`